oam/etc/Columnstore.xml

```diff
@@ -494,6 +494,7 @@
 			files are left behind.
 		<TempFilePath>/tmp/cs-diskjoin</TempFilePath>  -->
 		<TempFileCompression>Y</TempFileCompression>
+    <TempFileCompressionType>2</TempFileCompressionType>
```

Comment:
Wouldn't it be better to use symbolic names?

Reply:
Sure, thanks!